2-mandatory/3-shopping-list.js

```diff
@@ -19,7 +19,14 @@ let pantry = {
 };
 
```

Comment:
This looks almost perfect 😄 
One question: do you need the curly brackets `{` `}` on line 27?